pkg/deploy/api/types.go

```diff
@@ -311,6 +311,10 @@ type DeploymentConfigSpec struct {
 	// or failing. Post strategy hooks and After actions can be used to integrate successful deployment with an action.
 	Test bool
 
+	// Paused indicates that the deployment config is paused resulting in no new deployments on template
+	// changes or changes in the template caused by other triggers.
```

Comment:
I am disallowing `oc deploy --latest` for paused deployments. Regarding pausing the deployer, as per @smarterclayton's comments, we probably need to give it more thought (and work) so I guess the first version of `oc rollout pause` will not pause the deployer if there is one running.